docs/UserGuide.md

```diff
@@ -22,17 +22,19 @@ guest management tasks done faster than traditional GUI apps.
 
 1. Copy the file to the folder you want to use as the _home folder_ for your GuestBook.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the image below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
    * **`list`** : Lists all guests.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3`: Adds a guest named `John Doe` to the guest book.
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3 rq/Extra towels`: Adds a guest named `John Doe` to the guest book.
 
-   * **`edit`** `1` `n/Bob rc/no`: Edits the 3rd guest shown in the current list.
+   * **`edit`** `1` `n/Bob rc/no rq/Room Service`: Edits the 3rd guest shown in the current list.
```

Comment:
Should it be "Edits the first guest" instead of the third guest? Should we also spell it out since this is graded by the CS2101 side?

Reply:
Agreed!